specs/electra/beacon-chain.md

```diff
@@ -258,6 +256,15 @@ class PendingConsolidation(Container):
     target_index: ValidatorIndex
 ```
 
+#### `ValidatorRequests`
+
+```python
+class ValidatorRequests(Container):
+    deposit_requests: List[DepositRequest, MAX_DEPOSIT_REQUESTS_PER_PAYLOAD]  # [New in Electra:EIP6110]
```

Comment:
```suggestion
    deposits: List[DepositRequest, MAX_DEPOSIT_REQUESTS_PER_PAYLOAD]  # [New in Electra:EIP6110]
```

If we have `Requests` suffix in the struct, can we drop it from the fields definition?